menpo/image/base.py

```diff
@@ -444,7 +445,7 @@ def as_histogram(self, keep_channels=True, bins='unique'):
         >>>     plt.bar(centre, hist[k], align='center', width=width)
         """
         # parse options
-        if isinstance(bins, str):
+        if isinstance(bins, basestring):
```

Comment:
Can you explain why we need this? i.e. why can't we just isinstance str in both Py 2/3 for just looking for string input to methods